torch/testing/_internal/common_methods_invocations.py

```diff
@@ -17443,6 +17449,45 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
         op=lambda self, condition, other: refs.where(condition, self, other),
         supports_nvfuser=False,
     ),
+    PythonRefInfo(
+        "_refs.index_select",
+        torch_opinfo_name="index_select",
+        # empty_strided
+        supports_nvfuser=False,
+        skips=(
+            # no _refs support for Tensor.__setitem__
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref'),
+            # Sample out= with a stride of zero. This _out operation checks that the input has no
+            # inner overlap
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref_errors'),)
+    ),
+    PythonRefInfo(
+        "_refs.index_copy",
+        torch_opinfo_name="index_copy",
+        # empty_strided
+        supports_nvfuser=False,
+        skips=(
+            # no _refs support for Tensor.__setitem__
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref'),)
```

Comment:
ouch, so not a lot of testing for the new refs

Reply:
They are tested against aten via the aten executor, right? I had to fix very many edge cases for tensors of dimension 0 lol